examples/powertools-examples-core/terraform/infra/variables.tf

```diff
@@ -0,0 +1,14 @@
+variable "aws_access_key" {
```

Comment:
do we really need to have keys ??? can't it use the local aws config?

Reply:
Absolutely can be removed. It uses local aws config by default. These variables are only used if you want to override them with the Terraform CLI per example.
Going to remove them then